src/types/Stream.ts

```diff
@@ -1,15 +1,11 @@
-import { Profiles } from './Profile';
-
 export interface Stream {
-  arch: string;
-  context: string;
-  description: string;
-  end_date: string;
   name: string;
-  profiles: Profiles;
+  os_major: number;
+  os_minor: number;
+  os_lifecycle: string;
   start_date: string;
+  end_date: string;  
```

Comment:
If dates are sometimes undefined, good to add to type so we can catch it with type checking.